Configuration/Eras/python/Era_Run3_PbPb_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pbpb_run3_cff import pbpb_run3
+
+Run3_PbPb = cms.ModifierChain(Run3, pp_on_AA_2018, pbpb_run3)
```

Comment:
why not keep the same style as before and have "pp_on_AA" in the name[s]?

Reply:
pp_on_AA was intended to mean that we are running a pp-like reco on AA collisions.  This is in contrast to the dedicated heavy-ion reco, which was also still being maintained in 2018.  Now that such a dedicated reco sequence has been deprecated, I thought it best to simplify the naming.   I'm open to suggestion though.

Reply:
IIUC, the HeavyIon scenario still exists. So, there is still a clear case to still call this pp_on_AA. To me the naming should reflect the status of the release in a coherent way rather than a specific intention for the future.
I prefer a coherent migration instead of changing in a single point.

Also, would AA or HI still work, or does "PbPb" need to be used here?

Reply:
I don't think there's much potential for confusion of --scenario HeavyIons with pbpb_run3, whereas most people will not understand the meaning of pp_on_AA, but I don't insist on the "pp_on_X" part.   AA denotes generic ion collisions.  Up to now we had PbPb and XeXe.  If we ran these today, we would like use the same pp_on_AA modifier on both, so I used AA to distinguish from pA.   However, in Run 3 we are likely to have Oxygen-Oxygen collisions, for which we would use the standard pp high lumi reconstruction settings.  So pp_on_AA doesn't make a whole lot of sense.   Should I call it pp_on_PbPb_run3?  Or we just live with pp_on_AA_2021, in the name of continuity?

Reply:
I still think that ```pp_on_``` should still be a part of the era name. I'm fine to stay with PbPb if other options may need a different setup